src/components/VenueCard.js

```diff
@@ -3,17 +3,28 @@ import PropTypes from 'prop-types';
 
 export const VenueCard = ({ venueName, state, city, url, venuePhone, bookingEmail, bookingContact, PAStatus, venueComments}) => {
 
-  function validateEmail(email) {
+  const validateEmail = (email) => {
     const validate = /^(([^<>()\]\\.,;:\s@"]+(\.[^<>()\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
     return validate.test(email);
-  }
+  };
+
+  const isUrlValid = (Url) => {
+    const results = Url.match(/(http(s)?:\/\/.)?(www\.)?[-a-zA-Z0-9@:%._~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_.~#?&//=]*)/g);
+    if(results == null){
```

Comment:
Here you could try just using a return statement to check for true or false values. 
If you write 'return results !== null' it will give you 'true' or 'false' back. Or you could use a ternary, just an idea.